### ISSUE TITLE
Red and Blue color values swapped on iOS app (RoboVM issue?)

### ISSUE THREAD

**Issue by ajrod124** [libgdx/libgdx#3562]
To begin, I am not sure if this is a LibGDX issue, a RoboVM issue, or an issue with my application configuration. 

I have a major roadblock in releasing my iOS app, which concerns the swapping of the red and blue color  bits on my app. I have tested my app on an Android device, on both an OS X and Windows machines, and on the iOS simulator. On all these devices, the color values appear like the screenshot below:

![correct](https://cloud.githubusercontent.com/assets/9423063/11161959/86fc6316-8a41-11e5-92fd-3aa402da3ddf.png)

However, on an actual iOS device, the colors appear like the following image:

![incorrect](https://cloud.githubusercontent.com/assets/9423063/11161961/9c95c8de-8a41-11e5-81b0-59a9bd18c682.png)

I tested the app on multiple iOS devices running different versions of iOS (8 and 9), and it still appeared like the second, incorrect image. 

I tried to edit the color and depth formats in the IOSApplicationConfiguration to no avail, and tried all combination of the two. I also looked at the documentation for any pList attributes that may be causing the bug, but I found nothing of relevance. What could be a possible explanation to this error?

The project was first generated using the LibGDX Eclipse project generation utility back when LibGDX was still in version 1.5, and RoboVM was still in Beta. The projects have been converted to Gradle projects using the Eclipse Gradle plugin, and I am compiling the project using the Eclipse RoboVM plugin. Hopefully this information helps. I will provide code snippets of any part of my code if asked.

**Comment by Darkyenus** [comment]
For anybody in the future wondering what was (probably) the problem: iOS (the OS) likes "crunched" png images, so RoboVM provides function to crunch the images automatically. This crunching however violates the PNG standard, so when libGDX's png-compliant loader tries to load the images, they are "corrupted".

**TLDR**: Disable PNG crunch in RoboVM
```xml
<skipPngCrush>true</skipPngCrush>
```
